Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/clientcmd/api/helpers.go

```diff
@@ -31,6 +31,14 @@ func init() {
 	redactedBytes = []byte(string(sDec))
 }
 
+// IsConfigEmpty returns true if the config is empty.
+func IsConfigEmpty(config *Config) bool {
+	return len(config.AuthInfos) == 0 && len(config.Clusters) == 0 && len(config.Contexts) == 0 &&
```

Comment:
does `api.Semantic.DeepEqual(config, &Config{})` not work? That treats nil==map[]{}